src/main/java/com/google/cloud/teleport/templates/PubSubToSplunk.java

```diff
@@ -153,9 +154,11 @@
   
   private static final Boolean DEFAULT_INCLUDE_PUBSUBMESSAGE = false;
   
-  private static final String ATTRIBUTES = "attributes";
+  @VisibleForTesting
+  protected static final String ATTRIBUTES = "attributes";
+  @VisibleForTesting
+  protected static final String DATA = "data";
```

Comment:
Can we rename these constants to be more descriptive than generic `DATA` or `ATTRIBUTES`, e.g. `PUBSUB_MESSAGE_DATA_FIELD`

Reply:
Renamed to `PUBSUB_MESSAGE_DATA_FIELD`, `PUBSUB_MESSAGE_ATTRIBUTE_FIELD` and `PUBSUB_MESSAGE_ID_FIELD`